content/doc/book/system-administration/reverse-proxy-configuration-with-jenkins/reverse-proxy-configuration-apache.adoc

```diff
@@ -317,6 +318,163 @@ The _ProxyRequests Off_ prevents Apache from functioning as a forward
+systemctl restart jenkins
+
+# Verify the socket exists
+ls -l /var/run/jenkins/jenkins.socket
+
+# Ensure Apache can access the socket
+sudo chown jenkins:jenkins /var/run/jenkins/jenkins.socket
+sudo chmod 700 /var/run/jenkins/jenkins.socket
+
+# Create a new virtual host to proxy requests to the Unix socket
+nano /etc/apache2/sites-available/jenkins.conf
+
+# Add the following
+<VirtualHost *:80>
+    ServerName localhost
+
+    # Proxy to Unix domain socket
+    ProxyPass / unix:/var/run/jenkins/jenkins.socket|http://localhost/
+    ProxyPassReverse / unix:/var/run/jenkins/jenkins.socket|http://localhost/
+
+    # Preserve host header
+    ProxyPreserveHost On
+
+    # Logging
+    ErrorLog ${APACHE_LOG_DIR}/jenkins_error.log
+    CustomLog ${APACHE_LOG_DIR}/jenkins_access.log combined
+</VirtualHost>
+
+# Enable the site and restart Apache
+a2ensite jenkins.conf
+systemctl restart apache2
+
+# If Apache needs to share socket access with Jenkins, update its user/group
+nano /etc/apache2/envvars
+# Modify:
+export APACHE_RUN_USER=jenkins
+export APACHE_RUN_GROUP=jenkins
+
+# Restart Apache
+systemctl restart apache2
+
+----
+
+=== RHEL-Based Systems
+
+This 16 minute tutorial from Darin Pope configures  Unix Domain Sockets With Apache HTTP Server and Jenkins on AlmaLinux
+
+.Configure Using Unix Domain Sockets With Apache HTTP Server and Jenkins
+video::KxFJh4184vk[youtube, width=640, height=360]
+
+
+[source,bash]
+----
+# Ensure the modules are loaded by editing the module configuration
+dnf install httpd -y  # Install Apache if not already present
+
+# Allow HTTP and HTTPS traffic through the firewall
+firewall-cmd --zone=public --add-service=http --permanent
+firewall-cmd --zone=public --add-service=https --permanent
+firewall-cmd --reload
+
+# Edit the Jenkins systemd service to disable TCP and enable a Unix socket
+systemctl edit jenkins
+
+# Add the following override
+[Service]
+ExecStart=/usr/lib/jenkins/jenkins --pluginroot=/var/cache/jenkins/plugins --httpPort=-1 --httpUnixDomainPath=/var/run/jenkins/jenkins.socket
+
+
+# Reload and restart Jenkins
+systemctl daemon-reload
+systemctl restart jenkins
+
+# Verify the socket exists
+ls -l /var/run/jenkins/jenkins.socket
+
+# Ensure Apache can access the socket
+sudo chown jenkins:jenkins /var/run/jenkins
+sudo chmod 700 /var/run/jenkins
+systemctl restart jenkins
+
+# Create a new virtual host to proxy requests to the Unix socket
+mv welcome.conf welcome.old
+vi /etc/httpd/conf.d/jenkins.conf
+
+# Add the following
+User jenkins
+Group jenkins
+ProxyPass / unix:/var/run/jenkins/jenkins.socket|http://localhost/ nocanon
+ProxyPassReverse / unix:/var/run/jenkins/jenkins.socket|http://localhost/
+
+
+# To allow SE linux to traffic allow to the socket
+chcon -t httpd_sys_rw_content_t /var/run/jenkins/jenkins.socket
+
+
+# Restart Apache
+systemctl restart apache2
+
+----
+
+Access `http://localhost/` in a browser. If issues arise, check the logs:
+
+To check for any error logs:
```

Comment:
```suggestion
Once you have finished your configuration, access `http://localhost/` in a browser.
If you encounter any issues, check the logs for any errors with the corresponding commands:
```
Suggestion to adjust syntax and remove redundant lines (465 mimics the end of 463, this suggestion provides more context and direction).